scalacheck-binding/src/main/scala/scalaz/scalacheck/ScalazArbitrary.scala

```diff
@@ -492,7 +492,7 @@ object ScalazArbitrary {
     Gen.sized(fingerTree[A] _)
   }
 
-  implicit def IndSeqArbibrary[A: Arbitrary]: Arbitrary[IndSeq[A]] = Functor[Arbitrary].map(arb[List[A]])(IndSeq.fromSeq)
+  implicit def IndSeqArbitrary[A: Arbitrary]: Arbitrary[IndSeq[A]] = Functor[Arbitrary].map(arb[List[A]])(IndSeq.fromSeq)
```

Comment:
ABI breakage

Reply:
Dropped. I'll have to look into how scala expects to handle such things...